database/message.go

```diff
@@ -116,13 +117,21 @@ func (msg *Message) Scan(row Scannable) *Message {
 }
 
 func (msg *Message) Insert() {
-	_, err := msg.db.Exec("INSERT INTO message (chat_guid, guid, part, mxid, sender_guid, timestamp) VALUES ($1, $2, $3, $4, $5, $6)",
-		msg.ChatGUID, msg.GUID, msg.Part, msg.MXID, msg.SenderGUID, msg.Timestamp)
+	_, err := msg.db.Exec("INSERT INTO message (chat_guid, guid, part, mxid, sender_guid, timestamp, service) VALUES ($1, $2, $3, $4, $5, $6, $7)",
+		msg.ChatGUID, msg.GUID, msg.Part, msg.MXID, msg.SenderGUID, msg.Timestamp, msg.Service)
 	if err != nil {
 		msg.log.Warnfln("Failed to insert %s.%d@%s: %v", msg.GUID, msg.Part, msg.ChatGUID, err)
 	}
 }
 
+func (msg *Message) Update() {
+	_, err := msg.db.Exec("UPDATE message SET service=$1 WHERE mxid=$2",
+		msg.Service, msg.MXID)
+	if err != nil {
```

Comment:
Is storing the service necessary or is it just future-proofing?

Reply:
future-proofing, remnant from when I was trying to detect service changes. can be removed, probably not going to be needed.